.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--- Title your PR with a Jira reference (if available) followed by brief description - for example: "PAYARA-1234 Add readme file" -->
+
+## Description
+This is a bug fix / feature. <!-- delete/modify as applicable-->
+
+<!-- Provide some context here -->
+
+<!--- Please provide enough information here about the what and why of your change. Target for developers of any experience level to understand -->
+
+## Important Info
+
+### Dependant PRs (if applicable)
+<!--- Link any related or dependant PRs here with brief description why -->
+
+### Blockers (if applicable)
+<!-- Detail any blockers with links/info -->
+
+### Deadline (if applicable)
+<!-- Detail any deadlines, why? -->
+
+## Testing
+
+### New tests
+<!-- Link to the test suite PR or provide info -->
+
+### Testing Performed
+<!--- Please describe how you tested these changes.  -->
+
+### Test suites executed
+<!-- Which test suites did you run this against? put an 'x' in the appropriate box(s). Feel free to add others.-->
+- [ ] Quicklook
+- [ ] Payara Samples
+- [ ] Java EE7 Samples
+- [ ] Java EE8 Samples
+- [ ] Payara Private Tests
+- [ ] Payara Microprofile TCKs Runner
+- [ ] Jakarta TCKs
+- [ ] Microprofile Samples 1.2
```

Comment:
Do we actually want to include the 1.2 and 1.4 samples?
With us implementing MP 3.0 soon most of them simply won't work due to breaking changes in the specs.

Separate argument - shift them under Payara Samples

Reply:
removed